src/main/java/com/fasterxml/jackson/databind/deser/impl/BeanPropertyMap.java

```diff
@@ -151,25 +155,28 @@ private BeanPropertyMap(BeanPropertyMap src,
         }
         _hashArea[ix] = key;
         _hashArea[ix+1] = newProp;
+        _locale = locale;
     }
 
     @Deprecated // since 2.8
     public BeanPropertyMap(boolean caseInsensitive, Collection<SettableBeanProperty> props)
     {
-        this(caseInsensitive, props, Collections.<String,List<PropertyName>>emptyMap());
+        //TODO what to do here? can I change the constructor even though it's deprecated?
```

Comment:
What to do here? can I change the constructor even though it's deprecated?

Reply:
Odd, 3.0 should not have any deprecated ones, must have leaked via merge from 2.x (where addition of new constructor does lead to marking of old ones as deprecated). Signatures of deprecated methods should not be changed in most cases as they are strictly left for backwards compatibility (for old code that is compiled to access them).
But at the same time, deprecated methods need not support new features so using bogus (or default) `Locale` would be fine.